### DISCUSSION TITLE
[M-Z] (22 Gennaio 2019) Esercizio 3

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#434] [Esami & Esoneri M-Z]
## Traccia

<img width="932" alt="image" src="https://github.com/user-attachments/assets/0cdde988-ab8c-4d55-aa4a-9e358e046f4b">

>**Stesso esercizio:** #425 (15 Gennaio 2018 - Esercizio 3)

**Comment by rimaout** [comment]
<img width="700" alt="image" src="https://github.com/user-attachments/assets/073de08e-3afa-48a9-b3f1-e528243af207">

**Comment by matypist** [comment]
Questo esercizio è un duplicato di [[M-Z] (15 Gennaio 2018) Esercizio 3 #425](https://github.com/sapienzastudentsnetwork/metodi-matematici-per-l-informatica/discussions/425) , per evitare una frammentazione delle proposte di soluzione potete quindi scrivere/leggere eventuali possibili soluzioni direttamente lì
